data.tf

```diff
@@ -15,14 +15,34 @@ data "aws_iam_policy_document" "workers_assume_role_policy" {
   }
 }
 
-data "aws_ami" "eks_worker" {
+data "aws_ami" "eks_default" {
   filter {
     name   = "name"
     values = ["amazon-eks-node-*"]
   }
 
   most_recent = true
-  owners      = ["602401143452"] # Amazon
+  owners      = ["602401143452"]
+}
+
+data "aws_ami" "eks_gpu" {
+  filter {
+    name   = "name"
+    values = ["amazon-eks-gpu-node-*"]
+  }
+
+  most_recent = true
+  owners      = ["679593333241"]
```

Comment:
I don't know why it's different. It probably comes from a different team in AWS. The `amazon-eks-gpu-node` AMI is not open-source like the default EKS AMI. I got this account number from the AMIs listed here: https://docs.aws.amazon.com/eks/latest/userguide/gpu-ami.html

Reply:
@max-rocket-internet make sense. Probably, because Nvidia driver are not OpenSource. Maybe include the link `https://docs.aws.amazon.com/eks/latest/userguide/gpu-ami.html` in the documentation.

Reply:
^ This link is the right sort of comment to put in code where we otherwise would have no idea where the account owner came from.